src/main/java/com/singlestore/fivetran/destination/writers/LoadDataWriter.java

```diff
@@ -117,6 +117,11 @@ public void writeRow(List<String> row) throws Exception {
 
     @Override
     public void commit() throws InterruptedException, IOException, SQLException {
+        if (t == null) {
+            // nothing is written
+            return;
+        }
+
         outputStream.close();
```

Comment:
should you release outputStream anyway?

Reply:
no, outputStream is not created in this case